src/styles/blocks/header.scss

```diff
@@ -0,0 +1,50 @@
+html,
+body {
+  height: 100%;
+  margin: 0;
+}
+
+body {
+  background-image: url(../../images/header_main_image.png);
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
Here, this stylization should be applied to .header, not body, put them inside the .header stylization below.